spec/controllers/bookmarks_controller_spec.rb

```diff
@@ -35,9 +35,11 @@
       expect(body).to include(headers)
       data1.each do |value|
         expect(body).to include(value)
+        expect(body).to include("'9997412163506421'")
```

Comment:
@christinach I would expect this as a separate check from the set of values, or a change to the data.  Not a check within the check for each value.
Maybe this line https://github.com/pulibrary/orangelight/blob/50c9cc9e4ea17d102e033507c3f5abf233aaa114/spec/controllers/bookmarks_controller_spec.rb#L22. should just change to contain the quotes?

Reply:
@carolyncole I updated data id to include the single quotes